crates/viewer/re_view_spatial/tests/transform_clamping.rs

```diff
@@ -160,6 +160,35 @@ fn test_transform_clamping(base_transform: BaseTransform) {
                     ]),
                 )
         });
+
+        test_context.log_entity(
+            "base/transform_axes/more_transforms_than_axis_lengths",
+            |builder| {
+                builder
+                    .with_archetype(
+                        RowId::new(),
+                        TimePoint::STATIC,
+                        &re_types::archetypes::TransformAxes3D::new(1.0),
+                    )
+                    .with_archetype(
+                        RowId::new(),
+                        TimePoint::STATIC,
+                        &re_types::archetypes::InstancePoses3D::new()
+                            .with_translations([
+                                (-1.0, -1.0, 0.0),
+                                (1.0, -1.0, 0.0),
+                                (-1.0, 1.0, 0.0),
+                                (1.0, 1.0, 0.0),
+                            ])
+                            .with_scales([
+                                (1.75, 1.75, 1.75),
+                                (1.5, 1.5, 1.5),
+                                (1.0, 1.0, 1.0),
+                                (0.5, 0.5, 0.5),
```

Comment:
in the spirit of transform clamping it would be interesting to still test out the existing clamping semantics
i.e. if you have more translations than scales, we still apply clamping logic there since we treat the "poses" as a whole as a single unit and within that unit clamping applies